packages/babel-plugin/src/utils/normalize-props-usage.ts

```diff
@@ -108,6 +108,27 @@ const normalizeDestructuredString = (
   });
 };
 
+/**
+ * Create a member expression from a list of strings. For example, if
+ * `objectChain = ['a', 'b', 'c']`, the generated member expression will be
+ * `a.b.c`.
+ * @param objectChain List of strings
+ * @returns A member expression.
+ */
+const createNestedMemberExpression = (objectChain: string[]): t.Identifier | t.MemberExpression => {
+  if (objectChain.length === 1) return t.identifier(objectChain[0]);
+  if (objectChain.length > 1) {
+    const [init, last] = [
```

Comment:
is there a better name for all the items of a list except the last one? i looked it up and init (from haskell) was the only one that i could easily find

Reply:
In lodash it's `initial` as well IIRC; no clue why, but that feels the FP naming..